CLI.md

```diff
@@ -73,6 +73,7 @@
 		* [neuro config aliases](#neuro-config-aliases)
 		* [neuro config docker](#neuro-config-docker)
 		* [neuro config get-clusters](#neuro-config-get-clusters)
+		* [neuro config get-projects](#neuro-config-get-projects)
```

Comment:
should this command be under `admin` where all related commands are?

Reply:
ah, there is one already. missed it.
could you please remind why we need the two separate commands?

Reply:
There is already one in admin section, removed this from config.